analytics/src/analytics/cli.py

```diff
@@ -241,17 +241,17 @@ def show_and_or_post_results(
     """Optionally show the results of a metric and/or post them to slack."""
     # defer load of settings until this command is called
     # this prevents an error if ANALYTICS_SLACK_BOT_TOKEN env var is unset
-    from config import settings
+    from config import DBSettings
 
     # optionally display the burndown chart in the browser
     if show_results:
         metric.show_chart()
         print("Slack message:\n")
         print(metric.format_slack_message())
     if post_results:
-        slackbot = slack.SlackBot(client=WebClient(token=settings.slack_bot_token))
+        slackbot = slack.SlackBot(client=WebClient(token=DBSettings.slack_bot_token))
         metric.post_results_to_slack(
             slackbot=slackbot,
-            channel_id=settings.reporting_channel_id,
+            channel_id=DBSettings.reporting_channel_id,
             output_dir=Path(output_dir),
```

Comment:
Does this work? I believe you need to instantiate the object in order to use the variables otherwise you're just using the class definition itself.

Something like
```py
settings = DBSettings()
slackbot = slack.SlackBot(client=WebClient(token=settings.slack_bot_token))
```